main/wallet-api.md

```diff
@@ -0,0 +1,206 @@
+# Wallet API
+
+This document lists and describes the API commands available to the wallet.
+
+## Petnames
+
+The Agoric wallet uses *petnames* for issuers and purses.
+
+Petnames are your personal names for objects. No one else can see or
+modify a petname without your permission. You can think of them as
+your phone's contacts list. The actual phone number is what your phone
+uses to call someone, but for you to more easily tell who a number is
+associated with, you've assigned a petname to it, such as Mom,
+Grandpa, Kate S., etc.
+
+## Overview
+
+Wallet API commands work with the following object types:
+- `purse`: Stores assets until you withdraw them into a payment for use 
+- `issuer`: Creates empty purses and payments and maps minted assets
+  to them when assets are added or removed. Issuers verify and move
+  digital assets.
+- `offer`: Consists of what amount of what brand you're willing to
+  give, and what amount of what brand you want, as well as the
+  conditions under which the offer holder can cancel it.
+
+## Purse API Methods
+
+### `deposit()`
+- **Function:**
```

Comment:
this is different in format from the rest of our API docs. Why?

Reply:
Because I didn't like the other format with respect to it not clearly denoting which parameters were optional, no slot for errors, etc. When there's time, I want to write up a really good API template and convert everything to it.